src/lib/routes/admin-api/user-admin.ts

```diff
@@ -741,15 +741,15 @@ export default class UserAdminController extends Controller {
                 ),
             );
         }
-        const matrix = await this.accessService.permissionsMatrixForUser(
+        const overview = await this.accessService.getAccessOverviewForUser(
             user,
             project,
             environment,
         );
 
         // TODO add response validation based on the schema
         res.status(200).json({
-            matrix,
+            overview,
```

Comment:
Are we changing an API schema? Is it public? Just watching out for potential breaking changes

Reply:
Ok, maybe the matrix field is not "exposed" in our public schema. Probably done intentionally... actually the response body is not declared:

![image](https://github.com/user-attachments/assets/0f59092c-de31-44cb-a539-5384d383aab7)

Reply:
Good point, I'll add a task for this!